src/lab37/exercise4/index.js

```diff
@@ -0,0 +1,43 @@
+const cards = [
+  { value: 'A', visible: false, found: false },
+  { value: 'A', visible: false, found: false },
+  { value: 'B', visible: false, found: false },
+  { value: 'B', visible: false, found: false },
+  { value: 'C', visible: false, found: false },
+  { value: 'C', visible: false, found: false }
+];
+
+let visibleCard = null;
+
+function play(cards, index) {
+  handleFirstTry(cards, index);
+  handleSecondTry(cards, index);
+  console.log(cards);
+}
+
+function handleFirstTry(cards, index) {
+  // check if there is already a visible cards
+  if (visibleCard) return;
+
+  visibleCard = { ...cards[index], index: index };
+  cards[index].visible = true;
+}
+
+function handleSecondTry(cards, index) {
```

Comment:
I like that you divided it into two functions, it's clean and elegant.
I'd like to challenge to avoid the global variable `visibleCard` and have both functions work without sharing anything but the cards list.